pkg/worker/handler/stack/stack.go

```diff
@@ -57,65 +61,57 @@ type stack struct {
 // assigned the stack status 0.5, otherwise the unhealthy stack status 0 is
 // assigned.
 func (h *Handler) stack(det []detail) ([]stack, error) {
-	var err error
-
 	var sta []stack
+
 	for _, x := range det {
-		var inp *cloudformation.DescribeStackResourcesInput
+		var tag string
 		{
-			inp = &cloudformation.DescribeStackResourcesInput{
-				StackName: aws.String(x.arn),
-			}
+			tag = staTag(x.nam)
 		}
 
-		var out *cloudformation.DescribeStackResourcesOutput
-		{
-			out, err = h.cfc.DescribeStackResources(context.Background(), inp)
-			if err != nil {
-				return nil, tracer.Mask(err)
-			}
-		}
```

Comment:
The core of the problem that we were running into was based on the call made here. This was wrong because stack resources are apparently not what we have to look at. What we have to look at instead are the stacks and not its resources.